io.github.mmarco94.tambourine.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.mmarco94.tambourine
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: run.sh
+finish-args:
+  # TODO: use `xdg-music:ro` once the file portal integration works
+  - --filesystem=host:ro
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  # TODO: this shouldn't be used, but if I don't do this, the DBUS connection gets closed
+  # almost immediately because of "Invalid client serial"
+  - --socket=session-bus
```

Comment:
Did you check what's going on with `--log-session-bus`? This is a huge blocker.

Reply:
Yes :)

I found a bug in the library I use for accessing the DBus, see https://github.com/hypfvieh/dbus-java/issues/220

Now I have removed that permission